ui/src/app.js

```diff
@@ -284,6 +282,16 @@ function processData (data) {
   }
 }
 
+function translate (data, fetchArgs) {
+  try {
+    return fetchArgs
+      ? locale.translate(data).fetch(...fetchArgs)
+      : locale.translate(data).fetch()
+  } catch (error) {
+    console.error('Error while translating: ', error)
```

Comment:
Should this return the original, untranslated string? Or translated into default, English?